v2.0/configure-replication-zones.md

```diff
@@ -333,18 +351,11 @@ Apply zone configurations to corresponding partitions:
 $ cockroach zone set roachlearn.students_by_list.australia --insecure  -f australia.zone.yml
 ~~~
 
-### Create a Replication Zone for System Ranges
+### Edit the Replication Zone for a System Range
 
-In addition to the databases and tables that are visible via the SQL interface, CockroachDB stores additional data in what are called system ranges. There are three categories of system ranges for which replication zones can be set:
-
-Zone Name | Description
-----------|------------
-**.meta** | The "meta" ranges contain the authoritative information about the location of all data in the cluster. If your cluster is running in multiple datacenters, it's a best practice to configure the meta ranges to have a copy in each datacenter.
-**.liveness** | <span class="version-tag">New in v2.0:</span> The "liveness" range contains the authoritative information about which nodes are live at any given time. If the liveness range is unavailable, the entire cluster will be unavailable, so giving it a high replication factor is strongly recommended.
-**.system** | The ".system" zone config controls the replication of a variety of important internal data, including information needed to allocate new table IDs and track the health of a cluster's nodes.
-**.timeseries** | The "timeseries" ranges contain monitoring data about the cluster that powers the graphs in CockroachDB's admin UI.
+In addition to the databases and tables that are visible via the SQL interface, CockroachDB stores internal data in what are called system ranges. CockroachDB comes with [pre-configured replication zones](#pre-configured-replication-zones) for most of these ranges.
 
-To control replication for one of the above sets of system ranges, create a YAML file defining only the values you want to change (other values will not be affected), and use the `cockroach zone set <zone-name> -f <file.yaml>` command with appropriate flags:
+To control replication for a system range, create a YAML file defining only the values you want to change (other values will not be affected), and use the `cockroach zone set <zone-name> -f <file.yaml>` command with appropriate flags:
```

Comment:
Optional, but we may want to change "other values will not be affected" to "other values will be copied from the `.default` zone".

Reply:
Done.